test-cases/longevity/longevity-lwt-basic-24h.yaml

```diff
@@ -1,19 +1,50 @@
-test_duration: 1500
-prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml n=10000000 ops'(insert_query=1)' cl=QUORUM -mode native cql3 -rate threads=30" ]
-stress_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml ops'(lwt_update_one_column=1,lwt_update_two_columns=1)' cl=QUORUM duration=1440m -mode native cql3 -rate threads=20",
-             "cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml ops'(lwt_update_one_column=1,lwt_update_two_columns=1)' cl=QUORUM duration=1440m -mode native cql3 -rate threads=20"
-            ]
-stress_read_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml ops'(select=1)' cl=SERIAL duration=1440m -mode native cql3 -rate threads=20" ]
+test_duration: 360
+
+prepare_write_cmd:
+  - >-
+    latte run -f lwt_insert -P rows_per_partition=1000 -P partition_sizes="\"100:1\"" --rate 5000 --duration=10000000
+    data_dir/latte/lwt_load.rn
+
+
+
+stress_cmd:
+  # TODO: what loader instance should be used? Its not clear needs to be clarified
+  # TODO: check rates
+  - >-
+    latte run -f upd_first_part --consistency=SERIAL --threads=1 --rate=500 --concurrency=900 --duration=360m
```

Comment:
Must be `--serial-consistency=SERIAL`.
The `--consistency=SERIAL` will work only with selects.